themes-default/slim/views/config_notifications.mako

```diff
@@ -13,1831 +13,1951 @@ window.app = {};
+                    api: null,
+                    id: null
+                },
+                twitter: {
+                    enabled: null,
+                    notifyOnSnatch: null,
+                    notifyOnDownload: null,
+                    notifyOnSubtitleDownload: null,
+                    dmto: null,
+                    username: null,
+                    password: null,
+                    prefix: null,
+                    directMessage: null
+                },
+                trakt: {
+                    enabled: null,
+                    pinUrl: null,
+                    username: null,
+                    accessToken: null,
+                    timeout: null,
+                    defaultIndexer: null,
+                    sync: null,
+                    syncRemove: null,
+                    syncWatchlist: null,
+                    methodAdd: null,
+                    removeWatchlist: null,
+                    removeSerieslist: null,
+                    removeShowFromApplication: null,
+                    startPaused: null,
+                    blacklistName: null
+                },
+                email: {
+                    enabled: null,
+                    notifyOnSnatch: null,
+                    notifyOnDownload: null,
+                    notifyOnSubtitleDownload: null,
+                    host: null,
+                    port: null,
+                    from: null,
+                    tls: null,
+                    username: null,
+                    password: null,
+                    addressList: [],
+                    subject: null
+                },
+                slack: {
+                    enabled: null,
+                    notifyOnSnatch: null,
+                    notifyOnDownload: null,
+                    notifyOnSubtitleDownload: null,
+                    webhook: null
+                }
+            }
+        };
+    },
+    computed: {
+        stateNotifiers() {
+            return this.$store.state.notifiers;
+        },
+        traktNewTokenMessage() {
+            const { accessToken } =  this.notifiers.trakt;
+            return 'Get ' + accessToken ? 'New ' : ' ' + 'Trakt PIN'
+        },
+        traktIndexersOptions() {
+            if (!this.configLoaded) {
+                return;
+            }
+            const { traktIndexers } = this.config.indexers.config.main;
+            const { indexers } = this.config.indexers.config;
+            let options = [];
+            const validTraktIndexer = Object.keys(indexers).filter(k => traktIndexers[k])
+            return validTraktIndexer.map(indexer => {
+                return { text: indexer, value: indexers[indexer].id }
+            })
+        }
+    },
+    created() {
+        const { $store } = this;
+        // Needed for the show-selector component
+        $store.dispatch('getShows');
+    },
+    mounted() {
+        $('#testGrowl').on('click', function() {
```

Comment:
I hope these click handlers are also being converted?

Reply:
Meh its alot of work

Reply:
Should be enough just to copy the code to new functions called `testGrowl` for example and then on the element add `@click="testGrowl"`.

Reply:
Nah. Cause you also have to create a new variable for the kodi-Result label. Replace the the jquery get api calls, etc. But no prob. Ill do it.

Reply:
Or its only about the click handlers, and not about fully removing jquery?

Reply:
Just the click handlers, it halts the mounted until all of those finish attaching. Using vue methods is a lot faster.

Reply:
done